stdlib/public/core/CompilerProtocols.swift

```diff
@@ -972,3 +972,19 @@ public protocol _ExpressibleByFileReferenceLiteral {
 /// then Array would no longer be a _DestructorSafeContainer.
 public protocol _DestructorSafeContainer {
 }
+
+/// This marker protocol describes the category of types that support implicit
+/// copying. By default, all types in Swift implicitly support copying and thus
+/// conformance to `Copyable` is implicitly synthesized. To suppress the
+/// implicit conformance synthesis, write `~Copyable` on the type declaration:
+///
+///     struct FileDescriptor: ~Copyable { /* ... */ }
+///
+/// The reading of this type declaration is "FileDescriptor suppresses implicit
+/// Copyable conformance".
+@_marker public protocol Copyable {}
+
+/// For older compilers interacting with newer stdlibs, define the unavailable
+/// `_Copyable` marker protocol that they expect to find.
+@available(*, unavailable)
+typealias _Copyable = Copyable
```

Comment:
must be a public typealias! I also wonder if it being a typealias even works. to play it safe maybe just duplicate the old one right here:
```
/// This marker protocol represents types that support copying.
/// This type is not yet available for use to express explicit
/// constraints on generics in your programs. It is currently
/// only used internally by the compiler.
@available(*, unavailable)
@_marker public protocol _Copyable {}
```